scripts.js

```diff
@@ -13,50 +14,43 @@ addCopy.classList.add("fas", "fa-copy", "copy", "icon", "push");
 addDelete.classList.add("fas", "fa-times", "delete", "icon", "edit");
```

Comment:
I want to make a snarky comment on you using all these unnecessary classes, but I think I made enough for this review :slightly_smiling_face: 

_PS: Yeah I know they're coming from font awesome, but you're not using them, and, for that matter, you'd be better off writing your own..._